packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -1791,19 +1791,16 @@ export class ThinEngine {
 
         if (IsWindowObjectExist() && IsDocumentAvailable()) {
             // make sure it is a Node object, and is a part of the document.

```

Comment:
This comment in not in line with the code anymore.

Reply:
As it was merged I will fix it in a future PR